docs/workflow/sdk-web-overview.mdx

```diff
@@ -8,96 +8,126 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 import UseCaseContainer from "../../src/components/UseCaseContainer";
 import UseCaseCard from "../../src/components/UseCaseCard";
 
-The Geocortex Workflow for Web Applications SDK consists of a TypeScript project that eases development of Workflow activities and custom form elements. It produces a [Workflow activity pack](key-concepts.mdx#activity-packs) that can be consumed by [web applications that use Workflow](key-concepts.mdx#geocortex-workflow-hosts).
+The Geocortex Workflow for Web Applications SDK consists of a TypeScript project that eases development of custom Workflow activities and form elements. It produces a [Workflow activity pack](key-concepts.mdx#activity-packs) that can be consumed by [web applications that use Workflow](key-concepts.mdx#geocortex-workflow-hosts).
 
-## Download
+## Overview
 
-You can download the Workflow Activity SDK for TypeScript [here](https://apps.geocortex.com/workflow/latest/activity-sdk.zip).
+The Geocortex Workflow SDK provides a [development toolkit](https://github.com/geocortex/vertigis-workflow-sdk/) for building custom Workflow activities and form elements into a package that can be deployed to Geocortex Web called an activity pack. Once the activity pack has been developed, it can then be deployed to your SaaS or on-premises environment.
 
 ## Requirements
 
+-   Geocortex Workflow 5.18 or later if running on-premises.
 -   The latest LTS version of [Node.js](https://nodejs.org/en/).
-    -   Run `node -v` in the terminal to confirm the version.
--   [npm](https://www.npmjs.com/get-npm) which is included with Node.
+-   A code editor of your choice. We recommend [Visual Studio Code](https://code.visualstudio.com/).
 
-## Setup
+## Getting Started
 
-1. Open the `activity-sdk` folder found in the SDK in your favorite IDE. We recommend using Visual Studio Code for the best experience.
-1. Run `npm install` in the terminal of this folder to install the project's dependencies.
+1. Run `npx @vertigis/workflow-sdk create <library-name>` where `<library-name>` is the name of the directory that will be created in the current working directory
+    - For example: `npx @vertigis/workflow-sdk create test-activity-pack`
+1. Open the `library-name` directory in your favorite IDE. We recommend using Visual Studio Code for the best experience.
+1. Run `npm run generate` to create a new activity or form element.
```

Comment:
Can we add something like "Run `npm run generate` and follow the onscreen prompts to create..."

Reply:
Good call. Done.